mesh_segmenter/src/convex_hull.cpp

```diff
@@ -0,0 +1,83 @@
+#include <mesh_segmenter/convex_hull.h>
+
+ConvexHullGenerator::ConvexHullGenerator(){}  
+
+void ConvexHullGenerator::makeMesh(const std::string& input, pcl::PointCloud<pcl::PointXYZ>& inMesh)
+{
+  //expects a .ply file
+  pcl::PLYReader Reader;
+  Reader.read(input, inMesh); //populate inMesh
+  return;
+}
+
+
+void ConvexHullGenerator::cleanMesh(const pcl::PointCloud<pcl::PointXYZ>& outMesh, pcl::PolygonMesh& outMeshPoly)
+{
+  //find centroid coords by finding average x, y, z 
+
+  Eigen::Matrix< float, 4, 1 > mid;
+  int centroid_success = pcl::compute3DCentroid(outMesh, mid);
```

Comment:
I expect this line causes a warning since the centroid_sucess variable is set, but never used. Add some error checking by looking at the returned value.